package.json

```diff
@@ -2,19 +2,44 @@
   "name": "@api3/chains",
   "license": "MIT",
   "version": "1.4.0",
+  "author": "API3 DAO",
   "private": false,
-  "main": "src/index",
+  "main": "./dist/cjs/index.js",
```

Comment:
I did this for validator, but It caused us pain (validator published wrong, apparently due to some yarn bug on some versions).

I don't think you need this unless you want to ship different implementations (in other words, if we avoid fs and other node modules the code will work in both node and browser).